.github/workflows/release-tag.yaml

```
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          ref: ${{ github.ref }}
+          ref: ${{ github.ref_name }}
```

The previous run worked fine here https://github.com/mesosphere/konvoy-image-builder/runs/7437609688?check_suite_focus=true
```suggestion
          ref: ${{ github.ref}}
```

---

The `ref` here most likely needs to full ref not just the name.